Scripting/SetupScript.py

```diff
@@ -15,30 +15,50 @@
 import Startup
 
 
+
 bind("W", "player_input_forward")
+bind("LeftStickUp", "player_input_forward")
 bind("S", "player_input_backward")
+bind("LeftStickDown", "player_input_backward")
 bind("D", "player_input_right")
+bind("LeftStickRight", "player_input_right")
 bind("A", "player_input_left")
+bind("LeftStickLeft", "player_input_left")
 bind("Shift", "player_input_sprint")
+bind("LeftStickClick", "player_input_sprint")
 bind("Space", "player_input_jump")
+bind("XboxA", "player_input_jump")
 bind("Control", "player_input_crouch")
+bind("XboxB", "player_input_crouch")
 
 bind("WheelUp", "player_input_inventory_up")
+bind("XboxLB", "player_input_inventory_up")
 bind("WheelDown", "player_input_inventory_down")
+bind("XboxRB", "player_input_inventory_down")
 
 bind("MouseUp", "player_input_look_up")
+bind("RightStickUp", "player_input_look_up")
 bind("MouseDown", "player_input_look_down")
+bind("RightStickDown", "player_input_look_down")
 bind("MouseRight", "player_input_look_right")
+bind("RightStickRight", "player_input_look_right")
 bind("MouseLeft", "player_input_look_left")
+bind("RightStickLeft", "player_input_look_left")
 
 bind("R", "player_input_build_rotate")
+bind("XboxX", "player_input_build_rotate")
 bind("Q", "player_input_drop")
+bind("RightStickClick", "player_input_drop")
 
 bind("K", "player_position_reset")
+bind("XboxSelect", "player_position_reset")
 bind("T", "fly_toggle")
+bind("XboxY", "fly_toggle")
 
 bind("MouseOne", "player_input_primary_fire")
+bind("XboxRT", "player_input_primary_fire")
 bind("MouseTwo", "player_input_secondary_fire")
+bind("XboxLT", "player_input_secondary_fire")
 
 chunk_render_distance(10)
-fps_max(200)
+fps_max(200)
```

Comment:
I hate to nitpick as this really isn't important but it is good practice to always have a single blank line at the beginning of files (it is an old Unix convention) My previous comment on this section was that there was two. Apologies, I should have been more clear.

Reply:
I am such a dope I meant end :man_facepalming: 

I will merge and fix, I am very sorry :man_facepalming: